fleet/lib/snmp-exporter-pre/files/snmp.yml

```diff
@@ -0,0 +1,2055 @@
+# WARNING: This file was auto-generated using snmp_exporter generator, manual changes will be lost.
+auths:
+  lsst_v1:
+    community: '{{.snmpLsstCommunity}}'
+    security_level: noAuthNoPriv
+    auth_protocol: MD5
+    priv_protocol: DES
+    version: 1
+  lsst_v2:
+    community: '{{.snmpLsstCommunity}}'
+    security_level: noAuthNoPriv
+    auth_protocol: MD5
+    priv_protocol: DES
+    version: 2
```

Comment:
The `snmpLsstCommunity`  here is not defined in the externalSecret.

Reply:
As mentioned https://github.com/lsst-it/k8s-cookbook/pull/280#discussion_r1534779064, this is intentional and still results in a working configuration.